DotNet/Report/KafkaProducers/SubmitReportProducer.cs

```diff
@@ -67,9 +67,6 @@ public async Task<bool> Produce(ReportScheduleModel schedule)
 
             _submissionReportProducer.Flush();
 
-            schedule.SubmitReportDateTime = DateTime.UtcNow;
```

Comment:
This is fine, but do we want to capture a data point for when the submission was requested. Do we have a  use case for that and what was the original intent of this field.

Reply:
See user story LNK-3700 which aims to improve this, later.